### PR TITLE
Chore setup linter

### PR DESCRIPTION
**Pull request by 41hulk**
### What does this PR do?
• Configure Linters with Github Actions and Good Practices of GithubFlow

### Description of Task to be completed?
- Create branches with different base branches.
- Understand the concept of version control.
- Follow Gitflow.
- Open pull requests in GitHub.
- Write descriptive, easy to understand commit messages.
- Write short, detailed, easy to understand descriptions of pull requests.
- Write a descriptive and easy to understand README in business English for every project submitted.
- Demonstrate an ability to submit a project for code review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```
Comment (Deepakdanger) [inline index.html:3]: -[ ] Lets include CSS file in head to style the `h1` tag.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @41hulk, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 
- Good Commits ✔️ 

## Required Changes ♻️

- [x] There should be a README.md file that contains necessary details.
- [x] Pull Request should have a short summary describing introduced changes in more detail.
- [x] _Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (41hulk) [reply index.html:3]: Awesome, Let me add it

**Comment by 41hulk** [comment]
@Deepakdanger changes required are all fixed and Ready for a review

File: readm.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- Twitter: [@ntare_guy](https://twitter.com/ntare_guy)
+- LinkedIn: [Ntare Guy](https://www.linkedin.com/in/ntare-guy/)
+
+## 🤝 Contributing
+
+My coding Partners and Awesome Friends, Ivan Silva and Mengstu Fentaw, I appreciate your help and let's Rock this🚀
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```
Comment (Whoistolu) [inline readm.md]: - [x] Kindly fix the `README.md` file name so that it can be easily accessed.
The file name you currently have is `readm.md`

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @41hulk 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the inline comments under the review for sections to improve._


Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: readm.md (hunk 2 above)
Comment (41hulk) [reply readm.md]: @Whoistolu all Solved

**Review by Deepakdanger — APPROVED** [review]
Hi @41hulk, 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**